units/src/amount/signed.rs

```diff
@@ -467,27 +573,8 @@ impl FromStr for SignedAmount {
     }
 }
 
-impl TryFrom<Amount> for SignedAmount {
```

Comment:
I have a PR open that address this: https://github.com/rust-bitcoin/rust-bitcoin/pull/3769.  Could we just cherry-pick that commit?